parsers/config.py

```diff
@@ -199,6 +201,9 @@
     # 'MD->RO':     ENTSOE.fetch_exchange,
     # MK
     'MK->RS':     ENTSOE.fetch_exchange,
+    # NI
+    'CR->NI':     NI.fetch_exchange,
```

Comment:
I tend to sort alphabetically, meaning that `CR->NI` should be under`CR`. Same for `HN->NI` which should be under `HN`